templates/hugo/.platform.template.yaml

```diff
@@ -28,6 +28,7 @@ info:
       <p>Hugo is a static site generator written in Go, using Go's native template packages for formatting.</p>
   # A list of tags associated with the template.
   tags:
+  - Go
   - Hugo
```

Comment:
We can probably remove the Hugo tag while we're at it, since there's nothing else with it.